### DISCUSSION TITLE
react中引入jsmind，编辑的时候，input框内是空白的，回车确认之后，才会展示出来

### DISCUSSION THREAD

**Discussion by Ken199416** [hizzgdev/jsmind#596] [General]
![image](https://github.com/hizzgdev/jsmind/assets/50293486/801cec64-3ff4-4deb-b31a-5e0d7d1ee92f)
![image](https://github.com/hizzgdev/jsmind/assets/50293486/c9eab8f5-ebe5-4099-8e97-f9de2b88f61e)
如图所示，
配置如下：
let options = {
            container: "jsmind_container",// [必选] 容器的ID，或者为容器的对
            editable: true,//是否启用编辑 
            theme: "greensea",//主题 
            mode: "side",//显示模式，full主要用于编辑，side主要用于展示
            view: {
                line_width: 2, // 思维导图线条的粗细
                line_color: "#ccc", // 思维导图线条的颜色
                draggable: true,
                hide_scrollbars_when_draggable: true,
                node_overflow: 'wrap',
                zoom: {             // 配置缩放
                    min: 0.5,       // 最小的缩放比例
                    max: 2.5,       // 最大的缩放比例
                    step: 0.2,      // 缩放比例间隔
                },
            },
            // mode: 'side',// 显示模式，子节点只分布在根节点右侧
            shortcut: {
                enable: true,        // 是否启用快捷键
                handles: {},         // 命名的快捷键事件处理器
                mapping: {           // 快捷键映射
                    addchild: [45, 4096 + 13, 9], 	// <Insert>, <Ctrl> + <Enter>
                    addbrother: 13,    // <Enter>
                    editnode: 113,   // <F2>
                    delnode: 46,    // <Delete>
                    toggle: 32,    // <Space>
                    left: 37,    // <Left>
                    up: 38,    // <Up>
                    right: 39,    // <Right>
                    down: 40,    // <Down>
                }
            },
        };

**Comment by hizzgdev** [comment]
你好，你在 https://jsmind.online 上能复现这个问题吗？另外你用的浏览器是 chrome 还是其它的？如果方便的话，你可以在这个页面上 jsmind_container 之外添加一个 `<input type="text">` 看看输入内容时会不会有同样的问题。 @Ken199416

**Reply by Ken199416** [reply]
感谢大佬回复是的，加了一个input也是有这个问题，应该是样式有冲突，在ccs中jmnode.selected的样式的color也是白色的，也就是说在编辑是，字体的颜色是白色，input的输入框的背景也是白色，所以看样子像是不展示，我现在把jmnode.selected的color覆盖成了黑色，目前是可以了展示出来了

**Reply by hizzgdev** [reply]
我感觉有问题的样式应该是 input。虽然把 jmnode.selected 的color改成黑色能解决问题，但是页面上其它的input里的内容应该还是显示不出来吧。不过如果你这个页面只有在 jsmind 里用到 input 的话，这样修改也没问题。 👍
